### PR TITLE
Cqlmapper support

### PR DESCRIPTION
**Pull request by pacejackson**

### CODE REVIEW

File: baseplate_cookiecutter/{{cookiecutter.project_slug}}/README.md [hunk 1]
```diff
@@ -5,14 +5,21 @@ This is the {{ cookiecutter.project_slug }} service. It is built on [Baseplate].
 [Baseplate]: https://reddit.github.io/baseplate/
 
 {% if cookiecutter.integrations.sqlalchemy %}
-## Database Initialization
+## Sqlalchemy Database Initialization
```
Comment (spladug) [inline baseplate_cookiecutter/{{cookiecutter.project_slug}}/README.md]: "Relational" ?

File: baseplate_cookiecutter/{{cookiecutter.project_slug}}/{{cookiecutter.module_name}}/models/cql.py [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+from baseplate.context.cassandra import cluster_from_config
+
+from cqlmapper import columns, connection
+from cqlmapper.management import create_keyspace_simple, sync_table
+from cqlmapper.models import Model
+
+
+class MyModel(Model):
+
```
Comment (spladug) [inline baseplate_cookiecutter/{{cookiecutter.project_slug}}/{{cookiecutter.module_name}}/models/cql.py]: extra blank here

File: baseplate_cookiecutter/{{cookiecutter.project_slug}}/{{cookiecutter.module_name}}/models/cql.py [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+    id = columns.UUID(primary_key=True, default=uuid.uuid4)
+    name = columns.Text()
+    value = columns.Integer()
+
```
Comment (spladug) [inline baseplate_cookiecutter/{{cookiecutter.project_slug}}/{{cookiecutter.module_name}}/models/cql.py]: not enough blanks here

File: baseplate_cookiecutter/{{cookiecutter.project_slug}}/{{cookiecutter.module_name}}/models/sql.py [hunk 4]
```diff
@@ -1,3 +1,4 @@
+
```
Comment (spladug) [inline baseplate_cookiecutter/{{cookiecutter.project_slug}}/{{cookiecutter.module_name}}/models/sql.py]: extra blank here?

**Comment by pacejackson** [comment]
:haircut_man: Addressed feedback.  /models does have an `__init__.py`
https://github.com/reddit/baseplate-cookiecutter/pull/25/files#diff-c3a60d2f395bc3efc9cd8e49a2560c92

**Pull request merged**